Packs/AzureSentinel/ReleaseNotes/1_5_71.md

```diff
@@ -0,0 +1,6 @@
+
+#### Incident Fields
+
+##### Microsoft Sentinel Classification Reason
+
+- Updated the Microsoft Sentinel Classification Reason incident field to default to None if missing rather than "Inaccurate Data".
```

Comment:
Please change the RN to -  Fixed an issue that sentinal one Classification Reason fields would default to "Inaccurate Data in the case where ..